confluent_kafka/schema_registry/schema_registry_client.py

```diff
@@ -329,9 +329,13 @@ def register_schema(self, subject_name, schema):
         if schema_id is not None:
             return schema_id
 
+        request = {'schema': schema.schema_str,
+                   'schemaType': schema.schema_type,
```

Comment:
Converted to 

        # CP 5.5 adds new fields (for JSON and Protobuf).
        if len(schema.references) > 0 or schema.schema_type != 'AVRO':
            request['schemaType'] = schema.schema_type
            request['references'] = schema.references